filterpage.html

```diff
@@ -0,0 +1,396 @@
+                />
+                <label for="Mercedes-Benz">Mercedes-Benz</label>
+              </li>
+            </ul>
+          </div>
+
+          <div>
+            <ul>
+              <li class="checkbox-wrapper">
+                <input
+                  type="checkbox"
+                  id="Hyundia"
+                />
+                <label for="Hyundia">Hyundia</label>
+              </li>
+              <li class="checkbox-wrapper">
+                <input type="checkbox" id="Tata" />
+                <label for="Tata">Tata</label>
+              </li>
+              <li class="checkbox-wrapper">
+                <input type="checkbox" id="Kia" />
+                <label for="Kia">Kia</label>
+              </li>
+              <li class="checkbox-wrapper">
+                <input type="checkbox" id="Honda" />
+                <label for="Honda">Honda</label>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <div class="body-wrapper">
+        <div class="body-and-arrow-wrapper">
+          <h2 class="body-heading">Body Type</h2>
+          <img
+            src="icons8-more-than-48.png"
+            alt="Upwards arrow"
+            class="body-arrow"
+          />
+        </div>
+
+        <div class="body-list-wrapper collapse">
+          <div>
+            <ul>
+              <li class="checkbox-wrapper">
+                <input type="checkbox" id="SUV" />
+                <label for="SUV">SUV</label>
+              </li>
+              <li class="checkbox-wrapper">
+                <input
+                  type="checkbox"
+                  id="Hatchback"
+                />
+                <label for="Hatchback">Hatchback</label>
+              </li>
+            </ul>
+          </div>
+
+          <div>
+            <ul>
+              <li class="checkbox-wrapper">
+                <input type="checkbox" id="Sedan" />
+                <label for="Sedan">Sedan</label>
+              </li>
+              <li class="checkbox-wrapper">
+                <input
+                  type="checkbox"
+                  id="Compact SUV"
+                />
+                <label for="Compact SUV">Compact SUV</label>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <button class="apply-filters__btn">Apply Filters</button>
+    </div>
+
+    <script>
+      let selectedMakeFilter = [];
+      let selectedBodyFilter = [];
+      let openFilterButton = document.querySelector(".open-filter__btn");
+      let selection = document.querySelector(".selection-wrapper");
+
+      //onclick of the open filter button the firstpage hides and the next filter selection page opens
+      let openFirstPageWrapper = document.querySelector(
+        ".open-filter-page-wrapper"
+      );
+      openFilterButton.addEventListener("click", function () {
+        selection.classList.remove("hide");
+        openFirstPageWrapper.classList.add("hide");
```

Comment:
I think no need to hide the first page. We can simply make a popup of full height and width.

Reply:
yes done